src/utils/childrenUtils.js

```diff
@@ -0,0 +1,40 @@
+import _ from 'lodash'
+import { Children } from 'react'
```

Comment:
Was curious as to the utilities associated with `React.Children` found [documentation here](https://facebook.github.io/react/docs/top-level-api.html#react.children) and posting here for other's convenience.